app/controllers/authenticate_controller.rb

```diff
@@ -94,6 +94,17 @@ def authenticate_oidc
     authenticate(input)
   end
 
+  def authn_oidc_missing_service_id
+    # We don't need to log the backtrace as we didn't travel in the application
+    # code yet. Thus, we don't use 'handle_authentication_error' or 'log_error'
+    log_error(
+      Errors::Authentication::AuthnOidc::ServiceIdMissing.new,
+      LogMessages::Authentication::AuthenticationError,
+      false
+    )
+    raise Unauthorized
```

Comment:
The comment says:

> we don't use 'handle_authentication_error' or 'log_error'

but then we do use `log_error` in the next line.

Reply:
you are correct. it's a leftover from when i thought not to use it.

Reply:
fixed